### PR TITLE
fix(cve): update packages

### LINKED ISSUE #16: Fix json-schema CVE
image
From a run in a push that no longer exists (made a push to trigger CI) (but now exists after a force push)

Related to https://github.com/StatCan/daaas/issues/832

https://github.com/advisories/GHSA-896r-f27r-55mw

There are more details on testing and screenshots in the [PR](https://github.com/StatCan/kubeflow-pipelines/pull/18)

### PR DESCRIPTION
**Pull request by Jose-Matsuda**
hacky but seems to work. this is the same cve that is on the `branch-1.5.1`
Closes #16

### CODE REVIEW

**Comment by Jose-Matsuda** [comment]
possible funny business, so like in our `branch-1.5.1` I was testing using a kind cluster, as well as using the 
```
kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=1.2.0"
kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/env/platform-agnostic-pns?ref=1.2.0"
```

and then in `frontend` running `NAMESPACE=kubeflow npm run start:proxy` I observed the following
image

Where `asdaa` was on `stc-master` and `asd` was running on the `json-schema-cve` branch

**Comment by Jose-Matsuda** [comment]
Upon checking logs I can see
image

**Comment by Jose-Matsuda** [comment]
Am currently checking if the pipelines fail. I am using the `Tutorail DSL -Control Structures` pipeline, and a default experiment

On branch 1.5.1 the result is
image

and for the others
image

on branch 1.5.1 with the cve fix the result is the same.


image

File: frontend/Dockerfile [hunk 1]
```diff
@@ -13,7 +13,11 @@ COPY . .
 
 RUN cd  /frontend
 RUN npm update & npm cache clean --force
+## FORCE SOME THINGS
+RUN npm install --save json-schema
+RUN npm install --save jsprim
 RUN npm ci && npm run postinstall
+
```
Comment (wg102) [inline frontend/Dockerfile]: remove new line

File: frontend/Dockerfile [hunk 2]
```diff
@@ -13,7 +13,11 @@ COPY . .
 
 RUN cd  /frontend
 RUN npm update & npm cache clean --force
+## FORCE SOME THINGS
```
Comment (wg102) [inline frontend/Dockerfile]: Please write a significant message, possibly mentionning the cve or at least the ticket linked to it.

**Review by wg102 — CHANGES_REQUESTED** [review]

**Comment by Jose-Matsuda** [comment]
on 1.2.0 with no cve fix the experiments are different so can't be super sure. 
image
image
image


1.2.0 with cve fix
image
image

**Comment by Jose-Matsuda** [comment]
Again i'm no pipelines expert so it may be best to have more experienced people test this. If it completely breaks the rollback should not be difficult

**Review by wg102 — APPROVED** [review]

**Pull request merged**